glue_solar/instruments/iris/iris.py

```diff
@@ -29,7 +29,7 @@ def _parse_iris_raster(data, label):
     for window, window_data in data.items():
         for i, scan_data in enumerate(window_data):
             w_data = Data(label=f"{window.replace(' ', '_')}-scan-{i}")
-            w_data.coords = scan_data.wcs
+            w_data.coords = WCSCoordinates(scan_data.header)
```

Comment:
Why the change here?

Reply:
I made the change here because I would like to use `WCSCoordinates` as far as possible, since it is native to `glue`. `WCSCoordinates` is built on top of `WCS`, so inherits everything from `WCS`, and more.